domains/Accounts/Middleware/RedirectIfAuthenticated.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Domains\Accounts\Middleware;
+
+use Closure;
+use Illuminate\Contracts\Auth\Factory as Auth;
+
+class RedirectIfAuthenticated
+{
+    protected $auth;
+
+    public function __construct(Auth $auth)
+    {
+        $this->auth = $auth;
+    }
+
+    public function handle($request, Closure $next, $guard = null)
+    {
+
```

Comment:
Nit: extra space, here. Care to remove it?

Reply:
Yes, thanks!